lib/RESTClient.js

```diff
@@ -256,20 +256,15 @@ class RESTClient {
     }
 
     request(method, beginPath, log, params, callback) {
-        let next;
+        let next = callback;
```

Comment:
Yeah, I just wanted to show a minimal change to the code so that you guys can review quickly (and love it).
Now you're already aware of it, I will remove `next`.